### ISSUE TITLE
Add a progression bar when pulling images

### ISSUE THREAD

**Issue by upils** [godarch/darch#9]
A progression bar would be great when pulling images.
Currently a user may think darch is stuck.

**Comment by pauldotknopf** [comment]
Yup. I am using ```containerd```, and their ```ctr``` command has a nice progress indicator. It should be just a matter of copy/paste.

https://github.com/containerd/containerd/blob/master/cmd/ctr/commands/content/fetch.go
